paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -852,6 +842,7 @@ internal class DefaultFlowController @Inject internal constructor(
                     .build()
             val flowController = flowControllerComponent.flowController
             flowController.flowControllerComponent = flowControllerComponent
+            PaymentSheet.FlowController.instance = flowController
```

Comment:
I'm hoping we can find a better way to do this.

We also need to clear this if we aren't able to do it better.

Reply:
This has been removed